libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -440,6 +440,9 @@ class AC_AttitudeControl {
     // mix between throttle and hover throttle for 0 to 1 and ratio above hover throttle for >1
     float               _throttle_rpy_mix;
 
+    // Yaw feed forward percent to allow zero yaw actuator output during extreme roll and pitch corrections
+    float               _feedforward_scalar;
```

Comment:
maybe we should initialise this to 1.0f?  I suspect it doesn't matter functionally because attitude_controller_run_quat() should always be run but that code is a long way away and I wonder if at some point there's a chance that we could get to the rate_cotnroller_run() without having run attitude_controller_run_quat().